edx_repo_tools/dependabot_yml.py

```diff
@@ -36,8 +41,22 @@ def _add_elements(self):
         if not found:
             self.elements['updates'].append(self.yml_instance.load(value))
 
+
+    def _add_reviewers(self):
+        if self.reviewer:   
```

Comment:
Can you move this if condition to the call of this function to reduce extra indentation in this function?

Reply:
sure